### PR TITLE
Specify the shade of "green"

### PR DESCRIPTION
**Pull request by mavit**
This works around Emacs bug 66538 (https://debbugs.gnu.org/cgi/bugreport.cgi?bug=66538).

### CODE REVIEW

**Comment by tarsius** [comment]
Before doing anything I would want to see how this is handled upstream.

Some thoughts, assuming that things remain as they are:

1. It's not like I picked this specific shade of green, because it is better than others, but because *some* green makes sense to use. Depending on the theme, different shades would be more appropriate than others. The idea is that theme authors would pick the appropriate shade.
2. IMO even with a dark background (I tried #000000 and #333333), both shades look good/readable enough.

In other words, I tend to think, that things should remain as they are. "green" communicates better, that some shade of green should be used, than "#00ff00" does.

**Comment by tarsius** [comment]
> Before doing anything I would want to see how this is handled upstream.

Well, I don't think there's really anything they can do about it -- retroactively.

> In other words, I tend to think, that things should remain as they are.

I've decided to go with that.
